keel-artifact/src/test/kotlin/com/netflix/spinnaker/keel/artifacts/NpmArtifactSupplierTests.kt

```diff
@@ -101,21 +111,23 @@ internal class NpmArtifactSupplierTests : JUnit5Minutests {
         )
       }
 
-      test("supports NPM versioning strategy") {
-        expectThat(npmArtifactSupplier.supportedVersioningStrategy)
+      test("supports NPM version sorting strategy") {
+        expectThat(npmArtifactSupplier.supportedSortingStrategy)
           .isEqualTo(
-            SupportedVersioningStrategy(NPM, NpmVersioningStrategy::class.java)
+            SupportedSortingStrategy(NPM, NpmVersionSortingStrategy::class.java)
           )
       }
 
       test("looks up latest artifact from igor") {
         val result = runBlocking {
           npmArtifactSupplier.getLatestArtifact(deliveryConfig, npmArtifact)
         }
-        expectThat(result).isEqualTo(latestArtifact)
+        expectThat(result!!.version).isEqualTo(latestArtifact.version)
```

Comment:
can you chain these togther with a `.isNotNull()` and `.isEqualTo`()`? I don't like using `!!`